documents/src/pages/utils/cache.md

```diff
@@ -0,0 +1,133 @@
+<!-- 
+title: Local caching Utility
+location: ./utils/cache
+type: page
+layout: default
+-->
+
+::status-complete::
+
+# Local cache
+The utility provides provides an alternative way to store data on your local machine.
+There are 2 main classes that collaborate: wrapper and storage. LocalCache is high level class that wraps the selected storage to be asynchronous api.
+The second is storage class. Store data on the user's machine. The storages has a list of support as below:
```

Comment:
Does developer needs to be aware that there is storage class ? I don't see any code related to the class here. I think we only need to mention what is LocalCache class, how to instantiate it and its storage options.

Reply:
Updated by replacing Storage classes content to more detail on localcahe class.